_topic_map.yml

```diff
@@ -279,6 +279,8 @@ Topics:
         File: manual_upgrades
       - Name: Blue-Green Deployments
         File: blue_green_deployments
+      - Name: Operating System Upgrades
```

Comment:
"upgrade" to me suggests an OS version change... maybe say "update" everywhere? What do other people think?

Also, are there other places we can link to this new topic from? I feel like no one's going to notice it...